common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgCustomResourceStore.java

```diff
@@ -393,11 +393,10 @@ private static boolean isCustomResourceExistingInDifferentOrganization(
 		return (status == HttpStatus.BAD_REQUEST && message != null && message.indexOf(magicMessage) != -1);
 	}
 
-	private static boolean isCustomResourceActiveAttached(final HttpClientErrorException clientException) {
-		HttpStatus status = HttpStatus.valueOf(clientException.getStatusCode().value());
+	private static boolean isCustomResourceActiveAttached(final Exception clientException) {
 		final String magicMessage = "Resource type cannot be deleted as there are active resources attached to it";
 		String message = clientException.getMessage();
-		return (status == HttpStatus.BAD_REQUEST && message != null && message.indexOf(magicMessage) != -1);
+		return (message != null && message.indexOf(magicMessage) != -1);
```

Comment:
You can use `message.contains(magicMessage)`

Reply:
addressed